build.gradle

```diff
@@ -82,6 +82,21 @@ task serve << {
     java.lang.Thread.sleep(Long.MAX_VALUE);
 }
 
+task installReactUImodules(type: Exec) {
+    commandLine "npm", "install"
+}
+
+task buildReactUI(type: Exec) {
+    inputs.file("package-lock.json").withPathSensitivity(PathSensitivity.RELATIVE)
+    inputs.file("webpack.config.js").withPathSensitivity(PathSensitivity.RELATIVE)
+    inputs.dir("src/jsMain/js").withPathSensitivity(PathSensitivity.RELATIVE)
+    
+    outputs.dir("$buildDir/processedResources/js/main/webpack")
```

Comment:
I have no idea what this line does or if it is even working. I just followed along with the documentation posted here: https://guides.gradle.org/running-webpack-with-gradle/

Reply:
Not sure either honestly.